index.js

```diff
@@ -18,6 +18,24 @@ exports.getBalancedGame = function (players) {
     };
 }
 
+function classifyCards(cards) {
```

Comment:
Also, we should call this function before the `while` on the getBalancedGame in order to call it just once.